Tests/test_api/base.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from flask import Flask
+from flask_testing import LiveServerTestCase
+
+from kalliope.core import LifoManager
+from kalliope.core.ConfigurationManager import BrainLoader
+from kalliope.core.ConfigurationManager import SettingLoader
+from kalliope.core.Models import Singleton
+from kalliope.core.RestAPI.FlaskAPI import FlaskAPI
+
+
+class RestAPITestBase(LiveServerTestCase):
+
+    def tearDown(self):
+        Singleton._instances = {}
+        # clean the lifo
+        LifoManager.clean_saved_lifo()
+
+    def create_app(self):
+        """
+        executed once at the beginning of the test
+        """
+        # be sure that the singleton haven't been loaded before
+        Singleton._instances = {}
+        current_path = os.getcwd()
+        if "/Tests" in current_path:
+            full_path_brain_to_test = current_path + os.sep + os.pardir + os.sep + "brains/brain_test_api.yml"
+            self.audio_file = "files/bonjour.wav"
+        else:
+            full_path_brain_to_test = current_path + os.sep + "Tests/brains/brain_test_api.yml"
+            self.audio_file = "Tests/files/bonjour.wav"
```

Comment:
What do you think about extracting this to a local method and run it in the `setUp` function

Reply:
actually this part cannot be moved because it belongs to the create_app method which is a [Flask testing method](https://pythonhosted.org/Flask-Testing/). SetUp is not executed before this one.